frontend/src/views/MatchPage.js

```diff
@@ -11,6 +17,16 @@ function MatchPage(){
                 <button class="mp-button" id="green">YES</button>
                 <button class="mp-button" id="red">NO</button>
             </div>
+            <div class="toggleButton">
+            <button color="primary" onClick={toggle} class="mp-button" id="blue">Show Matches</button>
```

Comment:
The color="primary" might not be doing anything here since you assign it a background color in the css.